### PR TITLE
Readme: Add editor integration instructions

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,16 +111,62 @@ If you installed globally, run
...
+
+Integration in the prettier plugin for your favorite editor might not be working yet, see see the related issues for [VS Code](https://github.com/prettier/prettier-vscode/issues/395), [Atom](https://github.com/prettier/prettier-atom/issues/395) and [Vim](https://github.com/prettier/vim-prettier/issues/119).
+
+For the moment, you can set up prettier to run on save like this:
+
+### Atom
+
+Install [save-autorun](https://atom.io/packages/save-autorun) and create a `.save.cson` file in your project with the following content:
+
+```cson
```
Comment (evilebottnawi) [inline README.md]: cson?

File: README.md [hunk 2]
```diff
@@ -111,16 +111,62 @@ If you installed globally, run
...
 To test it out on a PHP file:
 
-- Clone this repository.
-- Run `yarn`.
-- Create a file called `test.php`.
-- Run `yarn prettier test.php` to check the output.
+* Clone this repository.
+* Run `yarn`.
+* Create a file called `test.php`.
+* Run `yarn prettier test.php` to check the output.
```
Comment (evilebottnawi) [inline README.md]: We need use prettier for readme :smile:

Comment (czosel) [reply README.md]: This change was actually done by prettier :smile:

Comment (evilebottnawi) [reply README.md]: hm, https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqABAYQDbTumACwEsBndAJzgAcJTiYIKBPAOgB0oMAlAVynQADZgEMKUQRxgBqHvyGjxkzhkxUR8dCPQAzYtnxgR2AwBMh8UjFbVC1ZTNXrN2vQfRGTcc4MvXb9hxc6HwCwmIC1FQwMMRwFARwVjZ2ggQQHoRwYADWBFnoELww1MVSKiHy4eLoUXAxcQl+Kfbpmdl5RPhFJWUgADQgENSx0KTIoGIUEADuAApiCOMoIgBuEMSmAyAARhQiufUAytQHxFAA5sgwFLxwg+ek8TBz+xcAtiLIOsZPgwBWpAAHgAhfaHGBHETvOAAGXOcG+v3uIFOFCeFGQIE+FByplmUG2UXOMAA6psiMgABwABkGUTocFJ+2oWLqGNWiMGVAAjrxiFRXiIPl8kD9sH8QE93sRrrcUfRLgYAIq8CDwJESlEwEQ7cmmSlIABMgxuIn05wumAg70+WKgeG2vCeABVdctxU8AL5eoA

Comment (czosel) [reply README.md]: Hm, strange. Maybe I have a custom config somewhere, let me see ...

Comment (czosel) [reply README.md]: Seems that vim-prettier is doing that. I reverted the change.

File: README.md (hunk 1 above)
Comment (czosel) [reply README.md]: Yeah, the plugin seems to use cson instead of json (coffeescript object notation)

**Comment by czosel** [comment]
Totally agree! Can you add that? I don't have PHPStorm :wink:

**Comment by evilebottnawi** [comment]
@czosel i don't have too :smile:

File: README.md (hunk 2 above)
Comment (czosel) [reply README.md]: I know what happened now. vim-prettier was running the prettier version inside our repository's `node_modules`, which is still 1.12 and not 1.13 - we should probably update it :wink:

**Comment by czosel** [comment]
:smile: I think @mgrip can help out. Shall we add it in another PR though?

**Comment by evilebottnawi** [comment]
@czosel yep, just create issue, otherwise we can forget this

**Comment by czosel** [comment]
:+1: Issue created, do you want to merge?

**Pull request merged**